src/Settings/settings.json

```diff
@@ -737,5 +737,14 @@
         "desc": "Open the old file when parsing an old problem URL",
         "type": "bool",
         "tip": "If a problem URL was set for a file path, when parsing that problem\nfrom competitive companion again, the old file will be opened."
+    },
+    {
+        "name": "Language",
```

Comment:
I think "Locale" or "Translation" is better. "Language" can be confusing with the programming language.

Reply:
> I have just reviewed all codes, haven't checked the translations yet.
> 
> Besides where I left suggestions, I think it would be better if we can get rid of `QCoreApplication::translate`, however, I don't know how to achieve that.

The reason I use QCoreApplication::translate is that not all classes are based on QObject, but tr can only be used in QObjects.